### ISSUE TITLE
responsive_utils.dart not included as an export in responsive_framework.dart

### ISSUE THREAD

**Issue by mstainer14** [Codelessly/ResponsiveFramework#181]
When developing for web, the `breakpointsLandscape` property of `ResponsiveBreakpoints.builder` does not work as intended because the default value of `_landscapePlatforms` in `responsive_breakpoints.dart` is set to the following:

```
  static const List<ResponsiveTargetPlatform> _landscapePlatforms = [
    ResponsiveTargetPlatform.iOS,
    ResponsiveTargetPlatform.android,
    ResponsiveTargetPlatform.fuchsia,
  ];
```
In order to overcome this we need to set the `landscapePlatforms` property of `ResponsiveBreakpoints.builder`. However, because responsive_utils.dart is not included as an export in the barrel file, we cannot access the `ResponsiveTargetPlatform` class directly but have to manually import the file `import 'package:responsive_framework/src/utils/responsive_utils.dart';` which gives us the following error.

```
Import of a library in the 'lib/src' directory of another package.
Try importing a public library that exports this library, or removing the import.dart[implementation_imports](https://dart.dev/lints/implementation_imports)

library package:responsive_framework/src/utils/responsive_utils.dart
```

Is it possible to add the `responsive_utils.dart` to the export file or move to a new location that is accessible through the import?

Many thanks

**Comment by mstainer14** [comment]
@rayliverified

**Comment by rayliverified** [comment]
Let's add it ASAP.


That's a serious issue. Can you open a PR and I'll add you as a contributor
too.

**Comment by mstainer14** [comment]
https://github.com/Codelessly/ResponsiveFramework/pull/182

**Comment by Juliotati** [comment]
@rayliverified  Any prevision on when this gets published?

**Comment by rayliverified** [comment]
Not yet. I need to bundle some other changes for publishing.

**Comment by rayliverified** [comment]
Thanks for catching this and the PR. 

Merged and released in v1.5.0.
